bootloader build fails on Windows 10 (japanese)

<!--
Welcome to the PyInstaller issue tracker! Before creating an issue, please heed the following:

1. This tracker should only be used to report bugs and request features / enhancements to PyInstaller
    - For questions and general support, use the discussions forum.
2. Use the search function before creating a new issue. Duplicates will be closed and directed to
   the original discussion.
3. When making a bug report, make sure you provide all required information. The easier it is for
   maintainers to reproduce, the faster it'll be fixed.
-->

<!-- +++ ONLY TEXT +++ DO NOT POST IMAGES +++ -->

## Description of the issue
the build command `python waf all -v` inf bootloader dir fails with the latest release:
https://github.com/pyinstaller/pyinstaller/releases/tag/v6.11.1

the error message is 
```
Build failed
 -> task in 'OBJECTS' failed with exit status 2 (run with -v to display more information)
```

This seems to be caused by non-ascii characters `˙` in `pyi_pythonlib.c`:
https://github.com/pyinstaller/pyinstaller/blob/6d529f42837b17cb808c08caa320ec3f693402d5/bootloader/src/pyi_pythonlib.c#L55

### Context information (for bug reports)

* Output of `pyinstaller --version`: ```v6.11.1```
* Version of Python: 3.11.8
* Platform: Windows 10 (ja-JP) <!-- e.g GNU/Linux (distribution), Windows (language settings), OS X, FreeBSD -->
* How you installed Python: scoop <!-- e.g. python.org/downloads, conda, brew, pyenv, apt, Windows store -->
* Did you also try this on another platform? Does it work there?
No.

### A minimal example program which shows the error

downloading zip from https://github.com/pyinstaller/pyinstaller/releases/tag/v6.11.1 , unzip it and
```
cd bootloader
python waf all -v
```

### Stacktrace / full error message


```
'all' finished successfully (0.000s)
'distclean' finished successfully (0.005s)
Setting top to                           : C:\Users\<user-name>\Downloads\pyinstaller-6.11.1\pyinstaller-6.11.1\bootloader 
Setting out to                           : C:\Users\<user-name>\Downloads\pyinstaller-6.11.1\pyinstaller-6.11.1\bootloader\build
Python Version                           : 3.11.8 (tags/v3.11.8:db85d51, Feb  6 2024, 22:03:32) [MSC v.1937 64 bit (AMD64)]
MSVC target(s)                           : ['x64', 'x86_amd64']
Checking for 'msvc' (C compiler)         : C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\CL.exe
Checking size of pointer                 : 8
Platform                                 : Windows-64bit-intel detected based on compiler
Build tests                              : disabled
Checking for header stdbool.h            : yes
Checking for function unsetenv           : no
Checking for function mkdtemp            : no
Checking for function dirname            : no
Checking for function basename           : no
'configure' finished successfully (1.887s)
'make_all' finished successfully (0.005s)
Waf: Entering directory `C:\Users\<user-name>\Downloads\pyinstaller-6.11.1\pyinstaller-6.11.1\bootloader\build\debug'
[ 1/26] Compiling zlib\zutil.c
[ 2/26] Compiling zlib\inffast.c
[ 3/26] Compiling zlib\adler32.c
[ 4/26] Compiling zlib\inftrees.c
[ 5/26] Compiling zlib\crc32.c
[ 6/26] Compiling zlib\inflate.c
[ 7/26] Compiling src\pyi_path.c
[ 8/26] Compiling src\pyi_apple_events.c
[ 9/26] Compiling src\pyi_launch.c
[10/26] Compiling src\pyi_splashlib.c
[11/26] Compiling src\pyi_global_win32.c
[12/26] Compiling src\pyi_pythonlib.c
zutil.c

[13/26] Compiling src\pyi_archive.c
[14/26] Compiling src\pyi_multipkg.c
[15/26] Compiling src\pyi_pyconfig.c
[16/26] Compiling src\pyi_main.c
[17/26] Compiling src\pyi_utils_win32_low_level.c
inffast.c

[18/26] Compiling src\pyi_global_posix.c
inftrees.c

[19/26] Compiling src\pyi_utils.c
crc32.c

[20/26] Compiling src\pyi_python.c
adler32.c

[21/26] Compiling src\pyi_utils_posix.c
inflate.c

[22/26] Compiling src\pyi_exception_dialog.c
pyi_apple_events.c

pyi_path.c

[23/26] Compiling src\pyi_utils_win32.c
[24/26] Compiling src\pyi_splash.c
pyi_splashlib.c

[25/26] Compiling src\main.c
pyi_launch.c

pyi_pythonlib.c
C:\Users\<user-name>\Downloads\pyinstaller-6.11.1\pyinstaller-6.11.1\bootloader\src\pyi_pythonlib.c(1): error C2220: 次の警告はエラーとして処理されます
C:\Users\<user-name>\Downloads\pyinstaller-6.11.1\pyinstaller-6.11.1\bootloader\src\pyi_pythonlib.c(1): warning C4819: ファイルは、現在のコード ページ (932) で表示できない文字を含んでいます。データの損失を防ぐために、ファイルを Unicode 形式で 保存してください。

pyi_global_win32.c

pyi_global_posix.c

pyi_pyconfig.c

pyi_utils_win32_low_level.c

pyi_multipkg.c

pyi_archive.c

pyi_utils.c

pyi_main.c

pyi_python.c

pyi_utils_posix.c

pyi_exception_dialog.c

main.c

pyi_splash.c

pyi_utils_win32.c

Waf: Leaving directory `C:\Users\<user-name>\Downloads\pyinstaller-6.11.1\pyinstaller-6.11.1\bootloader\build\debug'
Build failed
 -> task in 'OBJECTS' failed with exit status 2 (run with -v to display more information)
```

Please also see <https://github.com/pyinstaller/pyinstaller/wiki/How-to-Report-Bugs>
for more about what would use to solve the issue.

--- a/bootloader/src/pyi_pythonlib.c
+++ b/bootloader/src/pyi_pythonlib.c
@@ -52,7 +52,7 @@ pyi_pylib_load(struct PYI_CONTEXT *pyi_ctx)
      * the 'ar' archives can be used for both static and shared objects.
      * A shared library can be loaded from such an archive like this:
      *   dlopen("libpythonX.Y.a(libpythonX.Y.so)", RTLD_MEMBER)
-     * This means that if python library name ends with ˙.a˙ suffix, we
+     * This means that if python library name ends with '.a' suffix, we
      * need to change it into:
      *   libpython3.6.a(libpython3.6.so)
      * Shared libraries whose names end with .so may be used as-is. */

